app/services/importers/create_statements.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+# This class provides a mechnism to import genesis statements
+# It is not designed to keep statements in sync
+# For that should be done by hand
+# We only create/update version 0.0.1 for initial values
+# Manually create new version of contract if there are changes
+
+require "csv"
+
+module Importers
+  class CreateStatements
+    def call
+      LeadProvider.includes(:cpd_lead_provider).each do |lead_provider|
```

Comment:
are all ECF and NPQ lead providers moving to the next cohort in prod, or should this be custom and rely on lead providers and NPQ providers being in the new specific cohort?